Geospatial.md

```diff
@@ -162,3 +192,18 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Invalid geospatial values
+
+An invalid geospatial value refers to any of the following cases:
+
+* `null`: A null value in Parquet.
+* A non-`null` value that are encoded in a valid WKB or bounding box format 
+  but are not considered valid under this specification, including:
+  * `NaN`: Not a Number. For example, `POINT EMPTY` in WKB is represented by a 
+    `Point` with each ordinate value set to an IEEE-754 quiet NaN value.
+  * `Empty geometries`: Geometries explicitly marked as empty in WKB using 
+    indicators such as `numPoints`, `numRings`, or `numGeometries`. Examples 
+    include `LINESTRING EMPTY` or `POLYGON EMPTY`.
+  * `Out-of-bounds coordinates`: Values that fall outside the valid range 
```

Comment:
Are writers encouraged to drop X values when `Out-of-bounds coordinates` has been detected or it is the readers' choice to check this rule? Or both?

Reply:
I think the writers are required to drop X values when Out-of-bounds coordinates has been detected.

I clarified the readers behavior below